webui/react/src/e2e/models/components/NavigationSideBar.ts

```diff
@@ -52,21 +52,27 @@ export class NavigationSideBar extends BaseReactFragment {
     parent: this.#nav,
     selector: `a[aria-label="${'Workspaces'}"]`,
   });
-  readonly createWorkspace = new BaseComponent({
+  readonly createWorkspaceFromHover = new BaseComponent({
```

Comment:
there's actually two create workspace buttons, so i made this one a little more specific.